src/components/DepositWidget/index.tsx

```diff
@@ -37,17 +37,30 @@ interface WithdrawState {
 }
 
 const BalancesWidget = styled(CardWidgetWrapper)`
-  ${CardTable} > tbody > tr:not([class^="Card__CardRowDrawer"]) > td {
-    &[data-label='Token'] {
-      font-family: var(--font-default);
-      letter-spacing: 0;
-      line-height: 1.2;
-      flex-flow: row nowrap;
+  ${CardTable} {
+    > thead,
+    > tbody {
+      > tr:not([class^='Card__CardRowDrawer']) {
```

Comment:
Is `Card__CardRowDrawer` generated by styled-components? I'm not sure it won't change at some point without us noticing.
Would be better to use `> tr:not([class^='${CardRowDrawer or whatever}'])` if possible

Reply:
i think this was maybe when Michel was changing CSS, you're right. good catch